packages/cli/src/cliEntry.js

```diff
@@ -120,6 +120,15 @@ const addCommand = (command: CommandT, ctx: ContextT) => {
       opt.default,
     ),
   );
+
+  /**
+   * We want all commands (like "start", "link") to accept these flags, so we
+   * add them here instead of in every single command. This way they'll be
```

Comment:
Hmm... "we add them here instead of in every single command". In this context, "here" is "in every single command" actually.